lib/suspenders/generators/base.rb

```diff
@@ -0,0 +1,20 @@
+require "rails/generators"
+require_relative "../actions"
+
+module Suspenders
+  module Generators
+    class Base < Rails::Generators::Base
+      include Suspenders::Actions
+
+      def self.default_source_root
+        File.expand_path(File.join("..", "..", "..", "templates"), __dir__)
+      end
+
+      private
+
+      def app_name
+        Rails.application.class.parent_name.demodulize.underscore.dasherize
```

Comment:
It looks like `Rails.application` returns `app_class.instance`, so we could skip a step:

```suggestion
        Rails.app_class.parent_name.demodulize.underscore.dasherize
```